### PR TITLE
Fix/characters programs

### PR DESCRIPTION
**Pull request by mahubgt**

### CODE REVIEW

File: characters-program/programs/characters-program/src/instructions.rs [hunk 1]
```diff
@@ -18,8 +18,6 @@ pub struct CreateCharacter<'info> {
         payer = owner,
         seeds = [
             PREFIX.as_ref(),
-            owner.key().as_ref(),
-            authority.key().as_ref(),
```
Comment (lucasig11) [inline characters-program/programs/characters-program/src/instructions.rs:22]: Why did  you also remove the gameAuthority key from the seeds?

Comment (lucasig11) [reply characters-program/programs/characters-program/src/instructions.rs:22]: The owner field should also be removed from the character account.

Comment (eliagoris) [reply characters-program/programs/characters-program/src/instructions.rs:22]: Reject authority!!!!!!!! Embrace freedom

Comment (mahubgt) [reply characters-program/programs/characters-program/src/instructions.rs:22]: It was a mistake to remove the gameAuthority from the seeds, i will revert that change.

**Review by eliagoris — APPROVED** [review]

**Pull request closed without merging**